### PR TITLE
Dynconfig in schduler gets the cdn configuration from the directory

### PR DESCRIPTION
**Pull request by gaius-qi**
Signed-off-by: Gaius <gaius.qi@gmail.com>

## Description
- Add `getCDNFromDir` in schduler's dynconfig.
- Add `Dynconfig.CDNDirPath` in schduler's config.

## Related Issue
#303 

## Motivation and Context
Scheduler does not need to rely on manager service.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist:

- [x] Code compiles correctly.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: scheduler/config/config_darwin.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ import (
 
 var config = Config{
 	Dynconfig: &DynconfigOptions{
-		Type:       1,
+		Type:       "local",
```
Comment (jim3ma) [inline scheduler/config/config_darwin.go:11]: LocalSourceType

File: scheduler/config/config_linux.go [hunk 2]
```diff
@@ -8,7 +8,7 @@ import (
 
 var config = Config{
 	Dynconfig: &DynconfigOptions{
-		Type:       1,
+		Type:       "local",
```
Comment (jim3ma) [inline scheduler/config/config_linux.go:11]: LocalSourceType

File: scheduler/config/config_test.go [hunk 3]
```diff
@@ -39,7 +39,7 @@ func TestSchedulerConfig_Load(t *testing.T) {
 			},
 		},
 		Dynconfig: &DynconfigOptions{
-			Type:       1,
+			Type:       "local",
```
Comment (jim3ma) [inline scheduler/config/config_test.go:42]: LocalSourceType

**Review by jim3ma — APPROVED** [review]

**Review by jim3ma — APPROVED** [review]
LGTM

**Review by 244372610 — APPROVED** [review]

**Pull request merged**